mule-artifact-it/mule-packager-it/src/test/resources/additional-plugin-dependencies/dependency-with-y-as-test-jar/pom.xml

```diff
@@ -0,0 +1,21 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>group.id.y</groupId>
+    <artifactId>artifact-with-y-as-test-jar</artifactId>
+    <version>1.0.0</version>
+    <packaging>jar</packaging>
+    <name>Dependency</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>group.id.y</groupId>
+            <artifactId>artifact-id-y</artifactId>
+            <version>1.0.0</version>
+            <classifier>test-jar</classifier>
```

Comment:
I would have used another kind of classifier, anyways if it works for the scenario fine...